invenio_communities/members/services/service.py

```diff
@@ -466,7 +504,10 @@ def decline_invite(self, identity, request_id, uow=None):
         assert identity == system_identity
         member = self.record_cls.get_member_by_request(request_id)
         assert member.active is False
+        archived_invitation = ArchivedInvitation.create_from_member(member)
```

Comment:
Shouldn't this happen for all the [close](https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/customizations/request_types.py#L57) requests?